client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -178,7 +112,7 @@ class PublishingTools extends Component {
 				<SettingsSectionHeader title={ translate( 'Publishing Tools' ) } />
 
 				{ renderPostByVoice && this.renderPostByVoiceModule() }
-				{ renderPostByEmail && this.renderPostByEmailModule() }
+				{ this.renderPostByEmailModule() }
```

Comment:
I think we should put Post by Email above Post by Voice since it's used much more frequently.